File: WebApplication/src/modules/application-two/models/application-two-search-result-item.dto.ts
import { BankNomenclatureDto } from "src/modules/nomenclature/common/models/bank-nomenclature.dto";

export class ApplicationTwoSearchResultItemDto {
  id: number;
  bank: BankNomenclatureDto;
  totalSum: number;
  creditCount: number;
  creationDate: Date;
  recordDate: Date;
}

File: StudentCredit.Application/Applications/ApplicationsTwo/Dtos/ApplicationTwoSearchResultItemDto.cs
using StudentCredit.Application.Nomenclatures.Dtos;

namespace StudentCredit.Application.Applications.ApplicationsTwo.Dtos
{
	public class ApplicationTwoSearchResultItemDto
	{
		public int Id { get; set; }

		public BankNomenclatureDto Bank { get; set; }

		public decimal TotalSum { get; set; }

		public int CreditCount { get; set; }

		public DateTime CreationDate { get; set; }

		public DateTime RecordDate { get; set; }
	}
}
